Src/UIComponents/ModalWindow/ModalWindow.ts

```diff
@@ -0,0 +1,21 @@
+const modalWindow = (component: HTMLElement, closeWindow: () => void) => {
+    const backgroundDiv = document.createElement('div');
+    const modalWindowDiv = document.createElement('div');
+
+    backgroundDiv.appendChild(modalWindowDiv);
+    modalWindowDiv.appendChild(component);
+
+    backgroundDiv.className = 'modal-window-bg';
```

Comment:
All looks nice, just wonder why you used two ways of adding classes to elements.

Reply:
I checked if both work, and then I forgot to change 😭